en/docs/security-announcements/security-advisories/index.md

```diff
@@ -1,12 +1,12 @@
----
+in---
```

Comment:
Seems like this has been mistakenly added. WDYT?

Reply:
ah yes. wil remove. thanks!

Reply:
Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a